### PR TITLE
Updated content based on existing CoC and Google Form

### PR DESCRIPTION
**Pull request by paulsbruce**

### CODE REVIEW

File: _includes/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!---
+Sync with Google Doc: https://docs.google.com/document/d/1Y1hsaBLU24EfmqiLGQ9Ls3xrBEEEoqy6oOyQfVKHi8Q/edit
```
Comment (katebygrace) [inline _includes/code-of-conduct.md]: I think this should replace the google doc, not sync with -- github is a much better single source of truth

File: _posts/2000-01-01-intro.md [hunk 2]
```diff
@@ -5,24 +5,9 @@ color: black
...
-…it's focused on delivering information quickly, easily, configurably, and stylishly!
-
-Want to make a single-page site to show off something cool? Go [fork me on github!](https://github.com/t413/SinglePaged)
-
-<span id="forkongithub">
-  <a href="{{ site.source_link }}" class="bg-blue">
-    Fork me on GitHub
-  </a>
-</span>
+## Welcome to the public landing page for the Boston DevOps Slack group and meetup. Our community is a safe, confidential space for DevOps practitioners to chat and support each other.
```
Comment (katebygrace) [inline _posts/2000-01-01-intro.md]:
```suggestion
## Welcome to the public landing page for the Boston DevOps Slack group and meetup. Our community is a safe space for DevOps and related practitioners to chat and support each other.
```
We have nothing in the CoC on confidentiality and I don't think that can be expected on a public slack group - so suggest removing "confidential" and and "and related" to keep people like me (security AND sre) in ;)

File: _posts/2000-01-02-whos-invited.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Who's Invited"
+bg: blue
+color: white
+fa-icon: info
+---
+
+## You are welcome to join the Boston DevOps community if you’re currently working as a DevOps Engineering Manager, Team Lead, or any other role which involves managing people working in technology.
```
Comment (katebygrace) [inline _posts/2000-01-02-whos-invited.md]:
```suggestion
#### Who's invited? 

## You are welcome to join the Boston DevOps community if you are meaningfully connected to both devops and Boston. This includes DevOps Engineers/SREs, Engineering Managers/Team Leads, developers, and security engineers, or any other role which involves managing people working in technology. This does NOT include recruiters and marketers. 
```
Added a little specificity; might tweak a little more but wanted to be more verbose on this one

**Review by katebygrace — APPROVED** [review]
For the last picture; is there a logo that has the words devops on it? (as the icon for our slack does)
Otherwise, I put up some suggested changes but I have a lot more I want to tweak, but can do that in a separate PR

File: _includes/code-of-conduct.md (hunk 1 above)
Comment (paulsbruce) [reply _includes/code-of-conduct.md]: Agreed. Modified the doc to say so.

**Review by katebygrace — APPROVED** [review]

File: _posts/2000-01-02-whos-invited.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Who's Invited"
+bg: blue
+color: white
+fa-icon: info
+---
+
+#### Who's invited? 
+
+## You are welcome to join the Boston DevOps community if you are meaningfully connected to both devops and Boston. This includes DevOps Engineers/SREs, Engineering Managers/Team Leads, developers, and security engineers, or any other role which involves managing people working in technology. This does NOT include recruiters and marketers.
```
Comment (katebygrace) [inline _posts/2000-01-02-whos-invited.md]:
```suggestion
## You are welcome to join the Boston DevOps community if you are meaningfully connected to both Boston and devops. This includes DevOps Engineers/SREs, Engineering Managers/Team Leads, developers, and security engineers, anyone aspiring to towards devops, or any other role which involves managing people working in technology. This does NOT include recruiters and marketers.
```

One more rephrase sorry!

**Review by katebygrace — APPROVED** [review]

**Pull request merged**